web-ui/src/app/components/shared/ticker-panels/ticker.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+import { ITicker } from "./iticker";
+import {HttpClient, HttpErrorResponse, HttpHeaders} from "@angular/common/http";
+import { Observable } from "rxjs/Observable";
+import 'rxjs/add/observable/throw';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/do';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class TickerService {
```

Comment:
I would move this file to the `services` folder, what do you think?

Reply:
well, I see what you keep it here, let's not move it.

Reply:
I see you have created a tickerservice into services, I could migrate this methods there once you release your code